planning/behavior_velocity_planner/src/scene_module/occlusion_spot/occlusion_spot_utils.cpp

```diff
@@ -481,10 +466,15 @@ void generateSidewalkPossibleCollisionFromOcclusionSpot(
     lanelet::BasicPoint2d obstacle_point = {occlusion_spot_position[0], occlusion_spot_position[1]};
     lanelet::ArcCoordinates arc_coord_occlusion_point =
       lanelet::geometry::toArcCoordinates(path_lanelet.centerline2d(), obstacle_point);
+    const double dist =
+      std::hypot(base_point[0] - obstacle_point[0], base_point[1] - obstacle_point[1]);
+    // skip if absolute distance is larger
+    if (distance_lower_bound < dist) continue;
     const double length_to_col = arc_coord_occlusion_point.length - baselink_to_front;
     ArcCoordinates arc_coord_collision_point = {
       length_to_col,
       calcSignedLateralDistanceWithOffset(arc_coord_occlusion_point.distance, half_vehicle_width)};
+    // skip if occlusion is behind ego bumper
     if (length_to_col < offset_from_start_to_ego) {
       continue;
     }
```

Comment:
I think this judge should be in Line474.

Reply:
[a6ceb8c](https://github.com/autowarefoundation/autoware.universe/pull/354/commits/a6ceb8c20e4da72920718d7ce4b6ac1e1e2ce1e1)